src/RWGltf/RWGltf_GltfJsonParser.cxx

```diff
@@ -52,6 +53,245 @@ namespace
   private:
     Handle(NCollection_Buffer) myBaseBuffer;
   };
+
+  //! Helper class to parse "extras" section of glTF node.
+  //! In order to use, provide the ID and the "extras" value of the node
+  //! in the constructor, and then call Parse().
+  //! Alternatively, just call ParseExtras() static function.
+  class RWGltf_ExtrasParser
+  {
+  public:
+    //! Constructor. Initializes parser.
+    //! @param theParentID ID of the Json object that contains this "extras" value. Used only for printing messages.
+    //! @param theExtrasValue "extras" value to parse.
+    RWGltf_ExtrasParser(const TCollection_AsciiString& theParentID,
+                        const RWGltf_JsonValue& theExtrasValue);
+
+    //! Parses the "extras" value provided in the constructor.
+    //! @return Container with parsed data. May be nullptr if failed to parse.
+    Handle(TDataStd_NamedData) Parse();
```

Comment:
But the myResult allocated always. It is not related?
Can be it null? Or better to avoid allocate ahead?